semantics/src/main/scala/aqua/semantics/header/ModuleSem.scala

```diff
@@ -30,16 +31,18 @@ class ModuleSem[S[_]: Comonad, C: Picker](expr: ModuleExpr[S])(using
         name.value,
         shouldDeclare
       ),
-      (ctx, _) =>
+      (ctx, initCtx) =>
         // When file is handled, check that all the declarations exists
         if (declareAll.nonEmpty)
```

Comment:
What about declares all case? Is not the same logic needed here?
```aqua
aqua Lib declares *

import f from "..."
import g from "..."
```
seems like a valid use case